components/frontend/src/report/Report.js

```diff
@@ -33,8 +33,11 @@ export function Report({
 }) {
     function navigate_to_subject(event, subject_uuid) {
         event.preventDefault()
-        document.getElementById(subject_uuid).scrollIntoView()
-        window.scrollBy(0, 163) // Correct for menubar and subject title margin
+        const element = document.getElementById(subject_uuid)
+        if (element) {
+            element.scrollIntoView()
+            window.scrollBy(0, 163) // Correct for menubar and subject title margin
```

Comment:
Apparently, because it seems to work. Would be nice to be able to ask the menubar and title for their height, but I don't know how to do that. That rabbit hole is out of scope for this issue :-)